.github/workflows/update-third-party-notices.yml

```diff
@@ -0,0 +1,33 @@
+name: Check and Update Third Party Notices
+on: workflow_call
+
+jobs:
+  license-check:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
```

Comment:
nit: can we use a more recent node version?

Reply:
is node 18 good?